packages/vitest/README.md

```diff
@@ -0,0 +1,84 @@
+# ![@knapsack-pro/vitest](./img/logo.png)
+
+<h3 align="center">Speed up your tests</h3>
+<p align="center">Run your 1-hour test suite in 2 minutes with optimal parallelisation on your existing CI infrastructure</p>
+
+---
+
+<div align="center">
+  <a href="https://circleci.com/gh/KnapsackPro/knapsack-pro-js">
+    <img alt="Circle CI" src="https://circleci.com/gh/KnapsackPro/knapsack-pro-js.svg?style=svg" />
+  </a>
+</div>
+
+<br />
+<br />
+
+Knapsack Pro wraps [Vitest](https://vitest.dev/) and works with your existing CI infrastructure to parallelize tests optimally:
+
+- Dynamically splits your tests based on up-to-date test execution data
+- Is designed from the ground up for CI and supports all of them
+- Tracks your CI builds to detect bottlenecks
+- Does not have access to your source code and collects minimal test data
+- Enables you to export historical metrics about your CI builds
+- Replaces local dependencies like Redis with an API and runs your tests regardless of network problems
+
+## Getting Started
+
+1. Install the package
+
+```
+npm install @knapsack-pro/vitest
+```
+
+2. Update your CI to use a matrix build. See the [example workflow](.github/workflows/vitest-example-test-suite.yaml) for reference.
+3. Replace `vitest` with `@knapsack-pro/vitest` in your CI config
+4. Set the `KNAPSACK_PRO_TEST_SUITE_TOKEN` environment variable to your Knapsack Pro test suite token, and optionally set other environemnt variables as needed.
```

Comment:
Thanks for the review @ArturT 🙏 

Do you mind if I push back a little bit on this comment?  I agree it's consistent, but I find this approach a bit odd:

https://github.com/KnapsackPro/knapsack-pro-js/blob/1deb62943e8a4d999e54908c79db33ac2be87c0d/packages/jest/src/env-config.ts#L3-L6

Generally I don't think it's recommended to set env variables like this.  I asked v0 about this and it agreed: https://v0.dev/chat/fOuT0uzgMOL

<img width="831" alt="image" src="https://github.com/user-attachments/assets/2318ed00-f404-436d-8c39-1cb9f2780047" />

So I would suggest this instead:
```
if (process.env.KNAPSACK_PRO_TEST_SUITE_TOKEN_VITEST) {
  throw new Error("Please use KNAPSACK_PRO_TEST_SUITE_TOKEN instead of KNAPSACK_PRO_TEST_SUITE_TOKEN_VITEST");
}
```
That way if someone tries to use `KNAPSACK_PRO_TEST_SUITE_TOKEN_VITEST`, it will tell them to use `KNAPSACK_PRO_TEST_SUITE_TOKEN`

What do you think?

Reply:
> Do you mind if I push back a little bit on this comment? I agree it's consistent, but I find this approach a bit odd:
> ...

I agree with you. It's odd, and we don't like overriding the env var.

I'd like to break this into 2 separate steps.

For this PR, let's keep the old approach for consistency. Why it's important? 
We would like to optimize for the end user's convenience. Some users run 2 different test runners on CI. They might run RSpec and Vitest. They use:
* `KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC`
* `KNAPSACK_PRO_TEST_SUITE_TOKEN_VITEST`

The user can just add these to CI settings and that's it. For example, this is possible for CircleCI. 
Otherwise, the user would add 2 env vars with unique names to the CI settings, then redefine them in the CI yml file to set `KNAPSACK_PRO_TEST_SUITE_TOKEN`. This requires extra work from the user. We would like the user experience to be smooth.


I think the long-term approach should be to change how the `@knapsack-pro/core` package works so that we don't need to redefine env var like this. But I'd like to keep it outside of the scope of this PR. We keep internal notes for that. Is that fine with you?

Reply:
Sure, sounds good 👍